Makefile

```
@@ -38,12 +38,15 @@ clean_install: clean
 	rm -rf ~/.terraform.d/plugins/${HOSTNAME}/${NAMESPACE}
 
 .PHONY: lint
-lint: lint-go
+lint: lint-go lint-tf
 
 .PHONY: lint-go
 lint-go:
 	@test -z "$$(gofmt -l -d *.go | tee /dev/stderr)"
```

Could you please apply this suggestion and retry to pass the `Lint` check? I think that we the current one we are not catching erroneous files.

```suggestion
	@test -z "$$(gofmt -l -d ./.. | tee /dev/stderr)"
```

---

Yeah, I realised the issue too late. It was copied from a different place. I will fix this in a separate PR

---

https://github.com/maas/terraform-provider-maas/pull/103